README.md

```diff
@@ -3,9 +3,9 @@
 
 # OpenQA Public cloud Helper
 
-A web app which monitor, display and delete CSPs. It try to identify left over
+A web app which monitor, display and delete CSPs and linked resources. It try to identify left over
```

Comment:
```suggestion
A web app which monitors, displays and deletes CSPs and linked resources. It tries to identify left overs
```

Reply:
what I can say , I have grammar issues in my native language too :D . thanks !